lib/requirements.txt

```diff
@@ -4,3 +4,5 @@ opencv-python==4.4.0.44
 websockets==8.1
 Pillow==8.0.1
 requests==2.24.0
+tqdm==4.56.0
+matplotlib==3.3.3
```

Comment:
why matplotlib is added?

Reply:
ps for the requirements: maybe only need just one of pillow and opencv.

Reply:
In data_gen.py use rg_to_hsv and hsv_to_rgb
https://github.com/edgeai-neptune/neptune/blob/incremental-learning/examples/helmet_detection/training/data_gen.py#L7

Reply:
so if any extra requirements from example code, we need to update our lib?

Reply:
because the base image is provide by neptune system not by the user self.

Reply:
don't think so: the user can config the base image url. our lib requirements and base image is related, but different.

Think If we release our lib code which is then fixed, then the user can't import any extra package, this makes no sense.

Reply:
So each examples has a requirment, then user rebuild the images base our lib image. Or they just build a new images base `tensorflow/tensorflow` and add our the lib requirement?